learn/app-development/widgets/datalive/form.md

```diff
@@ -51,9 +51,9 @@ You can display form as a dialog. This means the form displays a pop-up when an
 
 To display form inline with a page. This is the default option.
 
-#### Align 
+#### Align
 
-Set the alignment of the form. You can choose from left, center, right alignment.
+Set the alignment of the form. You can choose from the left, center, right, and floating alignments.
```

Comment:
floating or angular material style is not part of 10.6.2, lets not put in these changes in the doc. @aparnasj @swethakundaram